packages/amesos2/src/Amesos2_EpetraRowMatrix_AbstractMatrixAdapter_def.hpp

```diff
@@ -246,6 +246,18 @@ namespace Amesos2 {
   const RCP<const Tpetra::Map<MatrixTraits<Epetra_RowMatrix>::local_ordinal_t,
                               MatrixTraits<Epetra_RowMatrix>::global_ordinal_t,
                               MatrixTraits<Epetra_RowMatrix>::node_t> >
+  AbstractConcreteMatrixAdapter<Epetra_RowMatrix, DerivedMat>::getMap_impl() const
+  {
+    // How to get the equivalent of Tpetra's getMap() returned (not just row vs col map)?? 
+    // is Map() correct (returns Epetra_BlockMap)
+    printf("Amesos2_EpetraRowMatrix_AbstractMatrixAdapter: Epetra does not have a 'getMap() method like Tpetra. Returning rcp(Teuchos::null).\n");
```

Comment:
Epetra does not seem to me to have a matching call like getMap() in Tpetra, which is what I needed to use for the gatherMap call. I'll figure out what the right map is to return in this case.